tests/http/kvmagent/test_destructive_storage.py

```
@@ -0,0 +1,125 @@
+
+import os
+import uuid
+
+import pytest
+
+pytestmark = [
+    pytest.mark.http,
+    pytest.mark.destructive,
+]
+
+TEST_STORAGE_PATH = "/tmp/ztest-storage-%s" % os.getpid()
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/ztest-storage-%s"

(S108)